content/faq/hybrid-application.md

```diff
@@ -35,6 +35,35 @@ await app.startAllMicroservicesAsync();
 await app.listen(3001);
 ```
 
+For bind `@MessagePattern()` only to one transport of microservices in hybrid application with multiple microservices can use second argument which should get value from `Transport` enum
```

Comment:
`with multiple microservices can use`

To

`with multiple microservices we can use`